tests/lib/rules/no-unknown-property.js

```diff
@@ -80,14 +80,8 @@ ruleTester.run('no-unknown-property', rule, {
     code: '<rect clip-path="bar" />;',
     output: '<rect clipPath="bar" />;',
     errors: [{message: 'Unknown property \'clip-path\' found, use \'clipPath\' instead'}]
-  }, {
-    code: '<script crossOrigin />',
```

Comment:
let's replace this with `crossorigin`, so that it tests that this is an invalid property?